git_buildpackage

```diff
@@ -163,9 +172,27 @@ def create_source_package(repo_dir, output_dir, revision, build_args):
     safe_run(command, cwd=repo_dir, interactive=True)
 
 
+    # Lets find the .dsc
+    #if not os.path.exists(
+    found_dsc = False
+    for fname in os.listdir(absoutput_dir):
+        if fnmatch.fnmatch(fname, '*.dsc'):
+            input_file = os.path.join(absoutput_dir, fname)
+            found_dsc = True
+            with open(input_file, 'a') as dsc_file:
+                dsc_file.write('Git-Id: ' + git_id)
+                dsc_file.write('Git-URL: ' + url)
```

Comment:
-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA256

Hi Daniel,

Okay, I'll create options for both, with default enabled.

Cheers!
Colin.

On 04/21/2016 08:08 AM, Daniel Gollub wrote:
> In git_buildpackage 
> <https://github.com/gollub/obs-service-git_buildpackage/pull/1#discuss
ion_r60534664>:
>
> 
>> @@ -163,9 +172,27 @@ def create_source_package(repo_dir,
>> output_dir, revision, build_args): safe_run(command,
>> cwd=repo_dir, interactive=True)
>> 
>> 
>> +    # Lets find the .dsc +    #if not os.path.exists( +
>> found_dsc = False +    for fname in os.listdir(absoutput_dir): +
>> if fnmatch.fnmatch(fname, '*.dsc'): +            input_file =
>> os.path.join(absoutput_dir, fname) +            found_dsc = True 
>> +            with open(input_file, 'a') as dsc_file: +
>> dsc_file.write('Git-Id: ' + git_id) +
>> dsc_file.write('Git-URL: ' + url)
> 
> I'm fine with having them all enabled by default. But the Git-URL
> just got created from the post-export hook.
> 
> — You are receiving this because you authored the thread. Reply to
> this email directly or view it on GitHub 
> <https://github.com/gollub/obs-service-git_buildpackage/pull/1/files/d
caaaf13ae8364b587058c1b4e7ffc17e55de7bd#r60534664>
>
-----BEGIN PGP SIGNATURE-----
Version: GnuPG v2

iQIcBAEBCAAGBQJXGI0GAAoJEFFTL+lu4Efo/VgP/14j6HWerWMcqLXOB0drY8qt
elVK3wOWm80ckRbTJQeoe9uskhldEw674ZNuzH9xZ+dgbui/rE9QKOQqYJrTlYVa
ARUNa9nKEcaM27H57Ezg4H+nM8F8VPsabV97FethoduFF/qPjKeDvKbDiBpsidaL
+OSUKASDcW5RLz0wkkpeDIYKdUzHDkyOgK0kYBHqHqsS7tXRgHKXL1qW7KyUE7BY
i5HqDIoruhTXc/aCpNrol2gvtDQFX3MGI09dT8pM/VXrs9BTrV0uSLDcDn686JYO
2kbaklw5Ob+O8zGk6fGwe++D9UyxnC8niCpJYytXzXgpRa88SAkSeeVP2z1R+0lw
aHVbEPc7MkNstAghgSdZd7lIZxI4NHxvepFzdBnh5E21Vixp9jcUxF6sMil0e+1l
ZqnlBSL1Fhza70I6auzaEy+BYHtYNHd3y2jqeU0zEuiWHhlL+s3ClagOv8h75Mt7
OK0jOtsWjWA9PIwLphLG2bex27BTDiUdKOofamo2V3Yxr/GXB1wZdkf2jkv9OBOY
C2tsqxYc+b2TAmssvWspdnLIP8DqdNutjyLAj78+ytnKEslz3wEWGks5Z+atPMAL
4V1fzWOt906v5Mv4TVlh78xmnAx4GEn5lCow4a7jKkFMFcXCLKRLp0o6S1b3G5Eo
YFv0/VEq0uPQ1nqZSYKG
=4PiK
-----END PGP SIGNATURE-----